modules/programs/antidote.nix

```
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+with builtins;
+with lib;
+let
+  module_config = config.programs.zsh.antidote;
```

The convention is to use `cfg` for the configuration associated with the module:

```suggestion
  cfg = config.programs.zsh.antidote;
```